### PR TITLE
remove sync version of InitializeService

### LINKED ISSUE #106: BigQueryConnection - get rid of InitializeService method

### PR DESCRIPTION
**Pull request by alexey-lin**
fix #106

### CODE REVIEW

File: BigQueryProvider/BigQueryConnection.cs [hunk 1]
```diff
@@ -56,15 +47,9 @@ public class BigQueryConnection : DbConnection, ICloneable {
...
-            CheckDisposed();
-            if(IsOpened)
-                throw new InvalidOperationException("Connection allready open");
-            try {
-                InitializeService();
-            }
-            catch(GoogleApiException e) {
-                state = ConnectionState.Broken;
-                throw e.Wrap();
+            try { OpenAsync().Wait(); }
```
Comment (Agitto) [inline BigQueryProvider/BigQueryConnection.cs]: @alexey-lin 
try to extract such functionality in to the static helper

Comment (alexey-lin) [inline BigQueryProvider/BigQueryConnection.cs]: @Agitto What for? This code is not so complicate and is not hardly duplicated.

File: BigQueryProvider/BigQueryConnection.cs [hunk 2]
```diff
@@ -29,16 +29,7 @@ public class BigQueryConnection : DbConnection, ICloneable {
 
         public override void ChangeDatabase(string databaseName) {
             CheckDisposed();
```
Comment (sady4850) [inline BigQueryProvider/BigQueryConnection.cs]: from the msdn:
IDbConnection.ChangeDatabase - Changes the current database for an open Connection object.

now it changes database regardless of ConnectionState

Comment (alexey-lin) [inline BigQueryProvider/BigQueryConnection.cs]: According to MSDN, the behavior in the case of a closed connection is not documented, so I believe there's no harm in doing this way. Moreover, before this change, the current database used to be changed in just the same way (with the only difference that the connection used to be opened).

**Pull request closed without merging**